src/20250120_Multi_FebruaryFundingUpdate/AaveV3Ethereum_FebruaryFundingUpdate_20250120.sol

```diff
@@ -0,0 +1,363 @@
+        ) - 1 ether
+      }),
+      ALC_SAFE
+    );
+  }
+
+  function _withdrawForMigration() internal {
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV2EthereumAssets.WETH_UNDERLYING,
+        amount: IERC20(AaveV2EthereumAssets.WETH_A_TOKEN).balanceOf(
+          address(AaveV3Ethereum.COLLECTOR)
+        ) - 1 ether
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV2EthereumAssets.WBTC_UNDERLYING,
+        amount: IERC20(AaveV2EthereumAssets.WBTC_A_TOKEN).balanceOf(
+          address(AaveV3Ethereum.COLLECTOR)
+        ) - 1e8
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV2EthereumAssets.LINK_UNDERLYING,
+        amount: IERC20(AaveV2EthereumAssets.LINK_A_TOKEN).balanceOf(
+          address(AaveV3Ethereum.COLLECTOR)
+        ) - 1 ether
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV2EthereumAssets.MKR_UNDERLYING,
+        amount: IERC20(AaveV2EthereumAssets.MKR_A_TOKEN).balanceOf(
+          address(AaveV3Ethereum.COLLECTOR)
+        ) - 1 ether
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV2EthereumAssets.USDC_UNDERLYING,
+        amount: AMOUNT_WITHDRAW_USDC
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV2EthereumAssets.USDT_UNDERLYING,
+        amount: AMOUNT_WITHDRAW_USDT
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+  }
+
+  function _withdrawAndSwapToETH() internal {
+    AaveV3Ethereum.COLLECTOR.withdrawFromV2(
+      CollectorUtils.IOInput({
+        pool: address(AaveV2Ethereum.POOL),
+        underlying: AaveV2EthereumAssets.DPI_UNDERLYING,
+        amount: IERC20(AaveV2EthereumAssets.DPI_A_TOKEN).balanceOf(
+          address(AaveV3Ethereum.COLLECTOR)
+        ) - 1 ether
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
+    AaveV3Ethereum.COLLECTOR.withdrawFromV3(
+      CollectorUtils.IOInput({
+        pool: address(AaveV3Ethereum.POOL),
+        underlying: AaveV3EthereumAssets.LUSD_UNDERLYING,
+        amount: IERC20(AaveV3EthereumAssets.LUSD_A_TOKEN).balanceOf(
+          address(AaveV3Ethereum.COLLECTOR)
+        ) - 1 ether
+      }),
+      address(AaveV3Ethereum.COLLECTOR)
+    );
+
```

Comment:
missing to withdraw aLUSD

Reply:
aLUSD was removed because there is no tokens available to withdraw. This has been reflected in the md with the removal of that table entry